service_mesh/v2x/ossm-route-migration.adoc

```diff
@@ -0,0 +1,17 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ossm-route-migration"]
+= Route Migration
+include::_attributes/common-attributes.adoc[]
+:context: route-migration
+
+toc::[]
+
+Automatic route creation, also known as Istio OpenShift Routing (IOR), is a deprecated feature that is disabled by default in {SMProductName} 2.5 and later versions. Migrating from IOR to explicitly-managed routes provides a more flexible way to manage and configure ingress gateways. When route resources are explicitly created they can be managed with other gateway and application resources as part of a GitOps management model.
```

Comment:
Automatic route creation, also known as Istio OpenShift Routing (IOR), is a deprecated feature that is disabled by default **for newly-created ServiceMeshControlPlanes** in {SMProductName} 2.5 and later versions.

Reply:
When route resources are explicitly created they can be managed **alongside** other gateway and application resources as part of a GitOps management model.

Reply:
both changes were made